staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -132,6 +136,11 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"The maximum number of mutating requests in flight at a given time. When the server exceeds this, "+
 		"it rejects requests. Zero for no limit.")
 
+	fs.DurationVar(&s.RequestTimeout, "request-timeout", s.RequestTimeout, ""+
```

Comment:
validate it as non-negative.

Reply:
Thanks. I've added the validation check.